scripts/generate_grpc_service_handlers.sh

```
@@ -5,17 +5,39 @@ TEMPLATES_DIR="templates"
+P2P_OUTPATH="internal/rpc/generated"
+LOCAL_OUTPATH="internal/rpc/generated"
+
+REGISTER_SERVICES_SERVER_FILE="${P2P_OUTPATH}/register_server_services.go"
+REGISTER_SERVICES_CLIENT_FILE="${P2P_OUTPATH}/register_client_services.go"
+UNMARSHALLING_FILE="${P2P_OUTPATH}/unmarshal.go"
+
+# TODO: figure out why claim imports typev instead of typev1 and
+DEFAULT_BLACKLIST="claim notification cancellation insurance" # we don't want to generate handlers for notifications - if we ever need more filters here the impl. need to change!
+
+# Valid values
+valid_types=("core" "product" "system")
```

Shellcheck (0.10.0)

[warning] valid_types appears unused. Verify use (or export if used externally).

(SC2034)